lib/spack/spack/build_systems/autotools.py

```diff
@@ -151,13 +151,25 @@ def configure(self, spec, prefix):
         options = ['--prefix={0}'.format(prefix)] + self.configure_args()
         inspect.getmodule(self).configure(*options)
 
+    def build_targets(self):
```

Comment:
Well, it's a minor point, but `configure_args` is a function because most of the time you'll need to code complex logic and we decided to avoid packagers having to write `@property` over and over. Make targets I think they will be 99% list of literal strings.